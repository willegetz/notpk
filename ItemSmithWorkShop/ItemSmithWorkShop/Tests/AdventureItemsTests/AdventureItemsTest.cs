using System;
using System.Collections.Generic;
using System.Linq;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using AdventureItems;
using ApprovalTests;
using ItemSmithWorkShop.AdventureItems.DisplayUtilities;

namespace AdventureItemsTests
{
	[TestClass]
	public class AdventureItemsTest
	{
		[TestMethod]
		public void TestCreateWeapon()
		{
			AdventureItemShop weaponShop = new WeaponItemSmith();
			AdventureItem weapon = weaponShop.OrderItem("Dagger");
			//weapon = new DisplayItem(weapon);
			Approvals.Approve(weapon.GetItem());
		}
	}
}
