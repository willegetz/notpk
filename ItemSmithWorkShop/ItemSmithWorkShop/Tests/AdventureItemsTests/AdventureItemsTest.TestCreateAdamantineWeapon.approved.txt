Adamantine Dagger:	'3002 gp'
Weight: '1.33 pound(s)'
To Hit: '+1'
Damage: '1d4'
	The dagger is a common secondary weapon. You get a +2 bonus on
	Sleight of Hand checks made to conceal a dagger on your body.
	Adamantine weapons are always considered to be masterwork quality.
	Adamantine bypasses a damage reduction of 20.